views/extras.tmpl

```diff
@@ -201,19 +201,24 @@
 {{define "addressTable"}}
   {{$TxnCount := .TxnCount}}
   {{$txType := .TxnType}}
+  {{$isMerged := .IsMerged}}
   {{if .Transactions}}
   <table class="table table-mono-cells table-sm striped" style="width: 100%;">
       <thead>
           <th>Tx Type</th>
           <th>Input/Output ID</th>
           {{if eq $txType "merged_debit"}}
               <th># of Input(s)</th>
-          {{else}}
-              <th class="text-right">Credit DCR</th>
-          {{end}}
-          {{if eq $txType "merged_debit"}}
+              <th class="text-center">Debit DCR</th>
+          {{else if eq $txType "merged_credit" }}
+              <th># of Outputs(s)</th>
+              <th class="text-center">Credit DCR</th>
+          {{else if eq $txType "merged" }}
+              <th>I/O&nbsp;Count</th>
```

Comment:
I suggest we put a `title` element on this to explain this is the number of inputs + outputs for this transaction that correspond to the address.